### ISSUE TITLE
Xcode 8

### ISSUE THREAD

**Issue by KeithMKolmos** [mwaterfall/MWPhotoBrowser#563]
Anyone know why this will compile in Xcode 7.3 but will not in Xcode 8?  I've tried clearing Derived Data but it shows a 

/Library/Developer/Xcode/DerivedData/MWPhotoBrowser-alquepoesrkvnebbewsbxrzidcii/Build/Products/Debug-iphonesimulator/MWPhotoBrowser.bundle: bundle format unrecognized, invalid, or unsuitable
Command /usr/bin/codesign failed with exit code 1

**Comment by youngsherlee** [comment]
I have same problem，but it can be run in the iPhone but simulator

**Comment by githubLittleCode** [comment]
maybe you can try this. product ->Scheme->Manage Schemes.  
Delete MWPhotoBrowser and add it again.

**Comment by youngsherlee** [comment]
It can't solve the error

**Comment by githubLittleCode** [comment]
Delete all things in manage Schemes and add again.I solve it by this.

**Comment by youngsherlee** [comment]
I can't solve it by your way，It's maybe due to the cocoapods format

**Comment by KeithMKolmos** [comment]
You are all correct it does work on the phone.  The issue is only in simulator.  I will try the suggestions here.

**Comment by cjwl0106** [comment]
I solved this problem, but do not know why to do so, will there be other bad effects?
![s](https://cloud.githubusercontent.com/assets/5037850/20086271/f2ed7004-a5a9-11e6-9d46-c4eee5ffcfcd.png)

**Comment by xilin** [comment]
Same problem here. It seems to be the `resource` folder name([ref](http://stackoverflow.com/a/29271922/535374))?

**Comment by youngsherlee** [comment]
It's maybe the reason of the resource folder,but the [ref](http://stackoverflow.com/questions/29271548/code-sign-error-bundle-format-unrecognized-invalid-or-unsuitable/29271922#29271922) is not working

**Comment by xilin** [comment]
Just update to pod 1.1.1 and re-run the `pod update`, it seems OK now.

**Comment by KeithMKolmos** [comment]
Was already running pod 1.1.1 so that doesn't seem to be the appropriate fix.
